setup.py

```diff
@@ -44,16 +44,16 @@
     install_requires=['django-model-utils>=4.0.0,<4.1.0'],
     extras_require={
         'qa': [
-            'black<=19.10b0',
+            'black<=20.8b1',
```

Comment:
See https://github.com/openwisp/openwisp-utils/issues/165.
Please set it back to the previous value.

Reply:
Okay, got i! 👍🏻

Reply:
Done!